### PR TITLE
docs(checkbox): modify use case

### PR DESCRIPTION
**Pull request by Aybrea**
## Checklist

List of tasks you have already done and plan to do.

- [x] Fix linting errors
- [x] Tests have been added / updated (or snapshots)

## Change information

- 调整Checkbox示例中「复选框组」下按钮位置
- 格式化Picker代码

### CODE REVIEW

File: packages/varlet-ui/src/checkbox-group/example/index.vue [hunk 1]
```diff
@@ -63,9 +63,9 @@ watchDarkMode(Themes.dark)
   <var-checkbox-group ref="group" v-model="value6">
     <var-checkbox :checked-value="0">{{ pack.eat }}</var-checkbox>
     <var-checkbox :checked-value="1">{{ pack.sleep }}</var-checkbox>
-    <var-button class="button" type="primary" @click="$refs.group.checkAll()">{{ pack.checkAll }}</var-button>
-    <var-button class="button" type="primary" @click="$refs.group.inverseAll()">{{ pack.inverseAll }}</var-button>
   </var-checkbox-group>
+  <var-button class="button" type="primary" @click="$refs.group.checkAll()">{{ pack.checkAll }}</var-button>
+  <var-button class="button" type="primary" @click="$refs.group.inverseAll()">{{ pack.inverseAll }}</var-button>
```
Comment (haoziqaq) [inline packages/varlet-ui/src/checkbox-group/example/index.vue:68]: Wrap with var-space

**Comment by haoziqaq** [comment]
I think the previous UI is better, and I suggest you only keep the picker formatting.

**Comment by chouchouji** [comment]
> I think the previous checkbox example UI is better, and I suggest you only keep the picker formatting.

[https://varlet.gitee.io/varlet-ui/#/en-US/checkbox-group](https://varlet.gitee.io/varlet-ui/#/en-US/checkbox-group)
image

**Comment by haoziqaq** [comment]
image

Too compact... I think that button should be at the bottom of the text and increase the gap size.

**Pull request merged**